test/quantization/fx/test_quantize_fx.py

```diff
@@ -5566,11 +5566,13 @@ def forward(self, x):
                 x = x.reshape()
                 return x
 
-        options = itertools.product([M1, M2], [True, False])
-        for M, is_qat in options:
+        for is_qat in [True, False]:
             m = M1().eval()
             example_inputs = (torch.randn(1, 3, 3, 3),)
-            m = prepare_fx(m, get_default_qconfig_mapping(), example_inputs=example_inputs)
+            if is_qat:
+                m = prepare_qat_fx(m, get_default_qat_qconfig_mapping(), example_inputs=example_inputs)
+            else:
+                m = prepare_fx(m, get_default_qconfig_mapping(), example_inputs=example_inputs)
```

Comment:
Is this related to flake8-bugbear? seems quantization-specific changes, not a lint

Reply:
Right, the original version of this test had an unused parameter `is_qat`, and this was my (possibly incorrect) attempt to guess what it was for. I can just remove the parameter if that would be better.

Reply:
We should not change any behavior in these linting PRs. 
In this case, you either want to remove the argument if BC-compatible. Otherwise silence and open an issue for the specific maintainer to fix it.

Reply:
OK, got it!